test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -743,8 +743,9 @@ def test_create_a_payment_intent_and_confirm
     assert get_response = @gateway.show_intent(intent_id, options)
     assert_equal 'requires_confirmation', get_response.params['status']
 
-    assert confirm_response = @gateway.confirm_intent(intent_id, nil, return_url: 'https://example.com/return-to-me')
+    assert confirm_response = @gateway.confirm_intent(intent_id, nil, options.merge(return_url: 'https://example.com/return-to-me', payment_method_types: 'card'))
```

Comment:
Since `payment_method_type` is optional what do you think about having a separate test instead of changing existing one?

Reply:
Absolutely! I'll add an extra remote test and have it check that the `payment_method` on the intent changes.

Reply:
New test is great! Maybe remove `payment_method_types` from this test that way we will know that it still works if `payment_method_types` are not provided.

Reply:
Done @tatsianaclifton !